src/collections/resources/articles/consul-service-mesh/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Service Mesh: Consul
```

Comment:
```suggestion
title: "Service Mesh: Consul"
```

Reply:
This was breaking the build.

Reply:
Oh okay, I'll keep that in mind for next time too.

Reply:
Speaking of next time... what do we have lined up? If nothing, let's get it lined up. 🙂

Reply:
Nothing is lined up as of now, but ready to get started with the next issue!